static/js/routes/common.min.js

```diff
@@ -7063,20 +7063,20 @@ return je(this,function(t,n,r){var i;return oe.isWindow(t)?t.document.documentEl
 
 }).call(this,typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {})
-},{"/Users/bradforda/Projects/cfgov-refresh/node_modules/jquery/dist/jquery.js":34}],72:[function(require,module,exports){
+},{"/Users/wilsong/repos/cfpb/cfgov-refresh/node_modules/jquery/dist/jquery.js":34}],72:[function(require,module,exports){
```

Comment:
> Also, what's the deal with a minified file having over 7000 lines?

This is because all site JS is currently being bundled together (including libraries). We had it set up to deliver two JS files, common.js and a page-specific JS bundle, which was split off from the browserify bundle using [factor-bundle](https://github.com/substack/factor-bundle). However, factor-bundle proved to be very unreliable when the team started playing with it and would commonly have EOF errors where it would half-write bundles and quit. Likely because of https://github.com/substack/factor-bundle/issues/59. It was removed in https://github.com/cfpb/cfgov-refresh/pull/507. The plan is to find a way to bring this functionality back and have a common bundle and page-specific bundle (likely by trying a browserify-shim alternative and adding factor-bundle back), which would eliminate things like having the slick-carousel JS included on every page when it's only used on /the-bureau/ pages. So anyway, that's the background to this.